control/preimage/src/main.rs

```diff
@@ -265,7 +265,7 @@ pub enum Format {
 struct Context {
     bridge_hub_api: Box<OnlineClient<PolkadotConfig>>,
     asset_hub_api: Box<OnlineClient<PolkadotConfig>>,
-    relay_api: Box<OnlineClient<PolkadotConfig>>,
+    _relay_api: Box<OnlineClient<PolkadotConfig>>,
```

Comment:
Why this change?

Reply:
Just suppress warning for variable unused.